src/domain/core/src/messages/core_message_types.rs

```diff
@@ -48,6 +55,7 @@ impl Message for DatasetLifecycleMessage {}
 pub struct DatasetLifecycleMessageCreated {
     pub dataset_id: DatasetID,
     pub owner_account_id: AccountID,
+    pub dataset_visibility: DatasetVisibility,
```

Comment:
Why don't we include the name of new dataset and the name of the owner account?

Reply:
It would likely be necessary at the very next step, where we integrate `DatasetEntry` and `MessagingOutbox`.

Reply:
> Why don't we include the name of new dataset ...

I was planning to add a dataset name to the message as part of the dataset ownership transfer implementation

> ... and the name of the owner account?

I don't see a case where we can use it yet, but no problem, it could be added